torch/testing/_internal/distributed/rpc/dist_optimizer_test.py

```diff
@@ -207,3 +207,5 @@ def test_dist_optim(self):
     def test_dist_optim_functional(self):
         self._test_dist_optim_base(optim.Adagrad, lr=0.05)
         self._test_dist_optim_base(optim.Adam, lr=1e-2, amsgrad=True)
+        self._test_dist_optim_base(optim.SGD, lr=0.05)
```

Comment:
Is this test now duplicated with the one on line 203 of this file?

Reply:
Yes you are right, right now we are defaulting to use the torchscripted version of optimizer, let me merge these two.